cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -513,7 +513,7 @@ def ensure_route_table_present(connection, module):
         except EC2ResponseError as e:
             module.fail_json(msg=e.message)
 
-    if propagating_vgw_ids is not None:
+    if propagating_vgw_ids:
```

Comment:
`propagating_vgw_ids` was *supposed* to be nullable, so you could use this module without needing to specify the propagating VGWs.

This changes (breaks!) that behaviour. Previous code that omitted `propagating_vgw_ids` will now have all propagating gateways deleted, instead of leaving them alone as intended. :fearful: